models/kernels/EtCalcFunction.hpp

```diff
@@ -0,0 +1,1115 @@
+#include <iostream>
+#include <cmath>
+#include <cstring>
+
+#define TRUE  1
+#define FALSE 0
+
+#define CP  1.006e+03  //  specific heat of air at constant pressure, J/(kg K), a physical constant.
+#define KV2 0.1681     //  von Karman's constant squared, equal to 0.41 squared, unitless
+#define TK  273.15     //  temperature in Kelvin at zero degree Celcius
+#define SB  5.67e-08   //  stefan_boltzmann_constant in units of W/m^2/K^4
+
+// NOTE: SET YOUR EDIT WINDOW TO 120 CHARACTER WIDTH TO READ THIS CODE IN ITS ENTIRETY.
+
+//#####################################
```

Comment:
I suggest following the Doxygen style comments that Chris added. Example here:
https://github.com/NOAA-OWP/ngen/blob/master/include/geojson/FeatureBuilder.hpp#L23